hoot-core/src/main/cpp/hoot/core/cmd/GenerateAlphaShapeCmd.cpp

```diff
@@ -81,10 +82,21 @@ class GenerateAlphaShapeCmd : public BaseCommand
       ShapefileWriter writer;
       writer.writePolygons(result, outputPath);
     }
+    else if (outputPath.toLower().endsWith(".geojson"))
+    {
+      OsmGeoJsonWriter writer;
+      Settings s;
+      //  Output the source tags in the geojson writer
+      s.set(ConfigOptions::getJsonOutputTaskingManagerAoiKey(), true);
+      writer.setConfiguration(s);
+      writer.open(outputPath);
+      writer.write(result);
+    }
```

Comment:
Use a `OsmGeoJsonWriter` setup specially for alpha-shape generation to output in the tasking manager format.